frontend/src/api/task.ts

```diff
@@ -0,0 +1,68 @@
+import { post } from "src/api/requests";
+
+/**
+ * Initializes the shape for the data being retrieved from the backend.
```

Comment:
```suggestion
 * Defines the "shape" of a Task object (what fields are present and their types) for
 * frontend components to use. This will be the return type of most functions in this
 * file.
```